containers/Dockerfile.android

```diff
@@ -1,38 +1,29 @@
-# TODO to significantly speedup this build make it multistage
-
-FROM ubuntu:20.04
+ARG HASH
```

Comment:
does it affect build caching to put this here vs right before its used?

Reply:
Nah it has to go here, doesn't effect the cache you can see when I fix the other comment that android is not rebuilt from scratch